src/transform/selecttransform.jl

```diff
@@ -1,25 +1,26 @@
 """
-    SelectTransform(dims::AbstractVector{Int})
+    SelectTransform(dims::Union{AbstractVector{Int}, AbstractVector{Symbol}})
 
-Select the dimensions `dims` that the kernel is applied to.
+Select the dimensions `dims` that the kernel is applied to. `dims` can be either all
+integers or symbols.
 ```
     dims = [1,3,5,6,7]
     tr = SelectTransform(dims)
     X = rand(100,10)
     transform(tr,X,obsdim=2) == X[dims,:]
 ```
 """
-struct SelectTransform{T<:AbstractVector{Int}} <: Transform
+struct SelectTransform{T<:Union{AbstractVector{Int}, AbstractVector{Symbol}}} <: Transform
```

Comment:
Could we maybe be even more general and just use
```suggestion
struct SelectTransform{T} <: Transform
```
?

Reply:
Sure, I'm happy to make it even more general!